cal/cal/static/js/services.js

```diff
@@ -238,10 +238,11 @@ analyticsApp.service('QueryService', function() {
     var ctrlDetails = [];
     var maxYValue = 0;
     var events = [];
+    var movingAverage = [];
     var xLabels = [];
     var yLabels = [];
-    for (var i = 0; i < data.length; i++) {
-      var event = data[i];
+    for (var i = 0; i < data[0].length; i++) {
```

Comment:
wait no cuz data[0] and data[1] after different lengths, so I can't iterate through both.. I guess I could iterate through data[0] as the length of data[0] will always be greater than or equal to data[1]...